File: application/src/main/java/hu/bsstudio/robonaut/Application.java
package hu.bsstudio.robonaut;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication(scanBasePackages = "hu.bsstudio.robonaut")
public class Application {

    public static void main(final String[] args) {
        SpringApplication.run(Application.class, args);
    }
}

File: server/src/main/kotlin/hu/bsstudio/robonaut/Application.kt
package hu.bsstudio.robonaut

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication(scanBasePackages = [ "hu.bsstudio.robonaut" ])
class RobonautServerApplication

fun main(args: Array<String>) {
    runApplication<RobonautServerApplication>(*args)
}
